### DISCUSSION TITLE
Throttle the mutation observer check

### DISCUSSION THREAD

**Discussion by ExileofAranei** [KingSora/OverlayScrollbars#640] [Q&A] [answered]
Greetings, thanks for the plugin, it is really awesome!
Is there a way to throttle the internal mutation observer callback (scrollbars is applied to `document.body`)?
I am building a site with a lot of gsap animations, webgl, lenis for smooth scrolling effect and etc. I ignored pretty everything what is animated to prevent scrollbars from being updated, but in performance tab it looks like scrollbars itself still does lots of calculations. I tried the `update.debounce`, but seems this works a bit in a different way. I am sure the height of my site does not changes with all of the mutations, so, for example, i want to strict the test to once per N seconds. Or maybe even to disable the built in mutation observer and take control over updating by myself

**Comment by KingSora** [comment, marked as answer]
Good day @ExileofAranei 

I believe this issue: https://github.com/KingSora/OverlayScrollbars/issues/637 and your issue have the same solution. Please check it out and give feedback :)

**Reply by ExileofAranei** [reply]
I've already ignored every unwanted mutations with `ignoreMutation` option, the documentation is pretty clear.
I fiind out the problem, it is not related to `OverlayScrollbars` library, sorry for wasting your time.
Misinterpreted the output of devtools performance tab, it seemed to me that mutation observer was the bottleneck, thats not true, my bad
